src/Models/News.php

```diff
@@ -22,8 +23,11 @@ class News extends Model
     use HasFactory;
 
     protected $fillable = [
+        "author_id",
+        "slug",
+        "title",
         "name",
-        "content",
+        "text",
```

Comment:
Why did you change this name? And why did you not change docblock?

Reply:
My mistake